sdks/python/apache_beam/io/external/xlang_parquetio_test.py

```diff
@@ -36,7 +36,7 @@
 PARQUET_WRITE_URN = "beam:transforms:xlang:test:parquet_write"
 
 
-@attr('UsesCrossLanguageTransforms')
+@pytest.mark.xlang_transforms
```

Comment:
Alternatively, you could remove the `no_xlang_transforms` marker and just comment out this line with the same TODO.

Reply:
I did this, I removed the `no_xlang_transforms` marker and commented out those tests

Reply:
A successful run with changes in marker `no_xlang_transforms ` https://ci-beam.apache.org/job/beam_PostCommit_XVR_Dataflow_PR/23/testReport/